kernel/rtlil.cc

```diff
@@ -47,6 +47,50 @@ int RTLIL::IdString::last_created_idx_ptr_;
 
 dict<std::string, std::string> RTLIL::constpad;
 
+const pool<IdString> RTLIL::builtin_ff_cell_types = {
```

Comment:
Is this an instance of the C++ Static Initialization Order Fiasco? (I think the textbook solution to this would be to put this as static member into a function that returns a const reference to it.)

Reply:
OK, does this look better?